locale/Makefile

```diff
@@ -41,7 +41,10 @@ uniq-po:
 		msguniq $$f -o $$f ; \
 	done
 
-tx-pull: $(EDITFILES)
+tx-new:
+	tx pull --minimum-perc 50 --all
```

Comment:
After thinking more about this, this is actually incomplete. The definition of EDITFILES is "take all PO files using `*/**/${DOMAIN}.po` and then substitute `.po` with `.edit.po`. This command only creates `$DOMAIN.edit.po` and then it's never picked up.

Slowly but surely I'm starting to figure this out.